helpers/__tests__/l10n-test.ts

```diff
@@ -17,32 +17,33 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-// require a "parent object" for spying
-import { requestMessages, resetBundle, translate, translateWithParameters } from '../l10n';
-import { getJSON } from '../request';
-
-jest.mock('../request', () => ({
-  getJSON: jest.fn(),
-}));
-
+/* eslint-disable camelcase */
```

Comment:
I removed this line and got no error when I ran eslint.

Reply:
This is definitely needed, there is a lot of translation keys that don't respect the camelcase rule in this file, we don't want to have them reported on SQ.

This rule raise warnings, not errors, so you won't see them when running `yarn lint`